ci/fireci/fireciplugins/binary_size.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import click
+import glob
+import json
+import logging
+import os
+import re
+
+from fireci import ci_command
+from fireci import gradle
+from fireci import prow_utils
+from fireci import uploader
+
+_logger = logging.getLogger('fireci.binary_size')
+
+
+@click.option(
+  '--pull-request/--no-pull-request',
+  default=False,
+  help='When running for pull requests, only size of affected SDKs are collected.'
+)
+@click.option(
+  '--log',
+  default=prow_utils.prow_job_log_link,
+  help='The link to the log of the prow job, which runs this coverage check.'
+)
+@click.option(
+  '--metrics-service-url',
+  envvar='METRICS_SERVICE_URL',
```

Comment:
Im trying to map where these values are populated, since there are no defaults and it does not seem to be wired in the prow configs either.

Reply:
It is actually defined as a preset in [prow configs](https://team.git.corp.google.com/firebase-android-core/test-infra/+/refs/heads/master/prow/config.yaml#118).